docs/eni-and-ip-target.md

```diff
@@ -0,0 +1,49 @@
+## `WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET` 
+
+The AWS VPC CNI has an [IPAM](https://en.wikipedia.org/wiki/IP_address_management) daemon (`aws-node`) running on
+every node that keeps track of all ENIs and IPs attached to the instance. The CNI binary (`/opt/cni/bin/aws-cni`) is
+invoked by the kubelet when a new pod gets added to the node. The binary calls `aws-node` asking for an IP for the pod.
+If there are no available IPs in the cache, it will return an error. The number of IPs to keep around is controlled by
+`WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET`. The default setting,`WARM_ENI_TARGET=1` means that `aws-node`
```

Comment:
I think it would just be best to replace all mentions of `aws-node` in this doc with `ipamd` and just state at the top of the doc that `ipamd` runs inside a Kubernetes Daemonset on each worker node.